bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala

```diff
@@ -59,7 +58,6 @@ trait ItcTable extends Table {
   peer.setRowSelectionAllowed(true)
   peer.setColumnSelectionAllowed(false)
   peer.setSelectionMode(ListSelectionModel.SINGLE_SELECTION)
-  peer.getTableHeader.setReorderingAllowed(false)
```

Comment:
Yes, I initially had forgotten to turn reordering back on, and the tooltip positions were maintained as they should be. I believe that the call to `getModelIndex` handles this.